### ISSUE TITLE
Developer Dependecies

### ISSUE THREAD

**Issue by klangner** [nwolverson/purescript-csv#8]
What do you think about moving some of the dependencies of this library into devDependencies?

At least testing frameworks:
```json
"devDependencies": {
    "purescript-quickcheck": "^3.1.1",
    "purescript-test-unit": "^10.1.0",
  }
```
It looks that there is no reason to pull them into the production binary.

**Comment by nwolverson** [comment]
Yup, those should be dev dependencies for sure. There should be no difference in built output (at least using psc-bundle) but dev-deps will not increase the dependency burden on any consumer of this library...

**Comment by nwolverson** [comment]
Fixed 9f73ea4a9845dccdfeda0c50be7c469903b12cba
